scripts/validateStellarAddresses.ts

```
@@ -0,0 +1,131 @@
+/**
+ * B-013: Validate existing stellar addresses in the database
+ * Run this script BEFORE applying the migration to ensure no invalid addresses exist
+ * 
```

ESLint

[error] Delete `·`

(prettier/prettier)